prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+
+const firstElement = languages[0];
+console.log('The first element of the array is ' + firstElement);
+
+const thirdElement = languages [2];
+console.log('The 3rd element of the array is', thirdElement);
+
+const length = languages.length;
+console.log('The length of the array is', length);
+
+const lastElement = languages[5];
```

Comment:
instead of using the number 5, you want to calculate the value making use of the length variable you defined on line 10. Let me know if you have any questions on that!

Reply:
got it! just fixed and pushed back here (-: